docs/src/digitising-process.md

```diff
@@ -1,184 +1,349 @@
 
-1. Create a QGIS project for Zimbabwe and import the four Zimbabwe sheets into the project.
+There are four map sheets which are provided.
+
+### Analysis of map sheets before georeferencing
+
+After loading the raster images into QGIS and doing some investigation. The following conclusion were established:
 
-2. Attempt to find any info about Zim projection, datum, etc. through research. Finding the memoir for the sheets would be the ideal situation. The maps being from 1986 mean that their projection system was most likely based on the Arc 1950 datum (which is based on the Clarke 1880 ellipsoid).
+* Attempt to find any info about Zim projection, datum, etc. through research. Finding the memoir for the sheets would
+   be the ideal situation. The maps being from 1986 mean that their projection system was most likely based on the Arc
+   1950 datum (which is based on the Clarke 1880 ellipsoid).
 
-3. Research did not yield the projection used for creating the sheets. The projection is required to make custom projected CRS for better georeferencing. The closest option found during research was a report from UNESCO from 1995 (Hydrogeological Maps A Guide and Standard Legend. Vol. 17., by Struckmeier, Wilhelm F, and Jean Margat) referencing the Zimbabwe Hydrogeological maps stating that "preferably UTM grid" was used for map locations. Using this information, a custom tmerc (Transverse mercator) projection system was made using the proj string:
+* Research did not yield the projection used for creating the sheets. The projection is required to make custom
+   projected CRS for better georeferencing. The closest option found during research was a report from UNESCO from
+   1995 (Hydrogeological Maps A Guide and Standard Legend. Vol. 17., by Struckmeier, Wilhelm F, and Jean Margat)
+   referencing the Zimbabwe Hydrogeological maps stating that "preferably UTM grid" was used for map locations. Using
+   this information, a custom tmerc (Transverse mercator) projection system was made using the proj string:
 
     ```
     Custom CRS: +proj=tmerc +lat_0=-19 +lon_0=30 +k=1 +x_0=0 +y_0=0 +a=6378249.145 +rf=293.4663077 +units=m +no_defs
     ```
 
-    The Central Meridian (lon_0) and the Latitude of Origin (lat_0) were taken from the Zimbabwe sheets and the **+a** and **+rf** values are for the Arc 1950 datum.
+   The Central Meridian (lon_0) and the Latitude of Origin (lat_0) were taken from the Zimbabwe sheets and the **+a**
+   and **+rf** values are for the Arc 1950 datum.
+
+#### Georefencing Parameters for the four sheets
+Georeferencing is an iterative process, and we needed to try different parameters to establish the most suitable ones 
+to use.
+
+The methods below describe the different parameters that were tested with the raster images.
+
+##### Test Parameters
 
-### Georeferencing Zimbabwe Sheets 1 through 4
+1. The first iteration of georeferencing Zimbabwe Sheet 1 was done using a `Thin Plate Spline transformation` using 11
```

Comment:
table will save us reading lot of text here

Reply:
![image](https://user-images.githubusercontent.com/100694811/191673920-3171d007-0c67-4a0c-9427-621008836ea2.png)